WordPress/gutenberg
# Move the HTML block into the blocks library package

In order to merge Gutenberg into Core easily, all the Gutenberg JavaScript Core must be shared a reusable npm package. This PR updates the HTML block to avoid relying on `wp.codeEditor` for now because there's no easy way to lazy-load WordPress scrips yet and move the `HTML` block into the blocks library.

The `wp.components.CodeEditor` has also been deprecated with this change.

**Testing instructions**

 - Make sure you can use HTML block properly.
 - There's no syntax highlighting for the moment but it's consistent with the Code Block and could be added later in a better way.
